test/sanity/validate-modules/validate-modules

```diff
@@ -60,6 +60,8 @@ else:
 BLACKLIST_DIRS = frozenset(('.git', 'test', '.github', '.idea'))
 INDENT_REGEX = re.compile(r'([\t]*)')
 TYPE_REGEX = re.compile(r'.*(if|or)(\s+[^"\']*|\s+)(?<!_)(?<!str\()type\(.*')
+SUBPROCESS_REGEX = re.compile(r'subprocess\.P.*')
+OS_CALL_REGEX = re.compile(r'os\.call.*')
```

Comment:
`call()` may be too generic for us to match on though.  We don't so that with `sys.exit`, may be ok.  `Popen` is not generic, so my comment for `Popen` is still valid.